### ISSUE TITLE
svg backend has limited encoding support with usetex

### ISSUE THREAD

**Issue by jkseppan** [matplotlib/matplotlib#8068]
### Bug report

**Bug summary**

As reported [on Launchpad](https://bugs.launchpad.net/ubuntu/+source/matplotlib/+bug/1438278), the combination of `text.usetex=True`, `text.latex.unicode=True`, and the svg backend can produce files with missing or wrong characters. A warning does get printed.

**Code for reproduction**

```python
from pylab import *
pp={'text.usetex': True, 'text.latex.unicode': True}
rcParams.update(pp)

plot(1,1)
ylabel(r'\textwon')
savefig('tmp.pdf')
savefig('tmp.png')
savefig('tmp.svg')
```

**Actual outcome**

The svg file lacks the won character. A warning like the following is printed:

```
../lib/matplotlib/textpath.py:389: UserWarning: The glyph (142) of font (/usr/local/texlive/2016/texmf-dist/fonts/type1/public/cm-super/sfss3583.pfb) cannot be converted with the encoding. Glyph may be wrong
  "be wrong" % (glyph, font_bunch.filename))
```

**Expected outcome**

The svg file should include the won character (₩), as the pdf and png files do. I don't think we document usetex as being supported in the svg backend, but for several use cases it works.

**Matplotlib version**

Current master (3d99e435d8b74216608f29adf97bafef7e64218c) installed from git with `pip install -e .`

**Comment by tacaswell** [comment]
Labeling this as hard as fixing it will likely involve understanding a whole lot about text encoding.

**Comment by slochower** [comment]
I believe I'm also experiencing this issue. In my case, I'm completely missing "fl" from a word in the SVG. I suspect this "fl" is a ligature in LaTeX and thus, possibly, treated as a special character (like the original report). In the generated SVG file, I can see
```
    <g id="text_26">
     <!-- Directional flux (cycle s$^{{-1}}$) -->
     <defs>
```
which is correct.

Using matplotlib version 2.1.2 installed via conda, with the following settings:
```python
    mpl.rc('text', usetex=True)
    mpl.rcParams['text.latex.preamble'] = [
        r'\usepackage{amsmath}',
        r'\usepackage{helvet}',
        r'\usepackage[EULERGREEK]{sansmath}',
        r'\sansmath',
        r'\renewcommand{\familydefault}{\sfdefault}',
        r'\usepackage[T1]{fontenc}',
        r'\usepackage{graphicx}'
    ]
```

![text8219-2](https://user-images.githubusercontent.com/5393995/37122118-1ae92386-2214-11e8-87f0-abdf1ef10be7.png)

**Comment by QuLogic** [comment]
This was fixed by #12928 + #14156.

**Comment by slochower** [comment]
Thanks for the update @QuLogic
